rocketpy/simulation/dispersion.py

```diff
@@ -279,308 +529,16 @@ def set_results(self):
                     self.results[key].append(value)
                 else:
                     self.results[key] = [value]
-        return None
 
     def set_processed_results(self):
         """Creates a dictionary with the mean and standard deviation of each
-        parameter available in the results
-
-        Parameters
-        ----------
-        None
-
-        Returns
-        -------
-        None
-        """
+        parameter available in the results"""
         self.processed_results = {}
         for result in self.results.keys():
             mean = np.mean(self.results[result])
             stdev = np.std(self.results[result])
             self.processed_results[result] = (mean, stdev)
```

Comment:
This is more pythonic and cleaner:

```suggestion
        for result, values in self.results.items():
            mean = np.mean(values)
            stdev = np.std(values)
            self.processed_results[result] = (mean, stdev)
```